projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php

```diff
@@ -2907,141 +2907,6 @@ function init() {
 	}
 }
 
-/**
- * Pocket sharing service.
- */
-class Share_Pocket extends Sharing_Source {
```

Comment:
Instead of deleting the class altogether, I would recommend leveraging our `Deprecated_Sharing_Source` class, which will allow you to remove the button for visitors, while still displaying an invitation for site admins to remove the now deprecated service.